packages/ui/src/components/va-time-input/_variables.scss

```diff
@@ -1,4 +1,5 @@
 :root,
 :host {
   --va-time-input-min-width: var(--va-form-element-min-width);
```

Comment:
I'm also not sure if `--va-time-input-min-width` is used somewhere, so this can be removed too likely.

Reply:
let me check, I will remove it if it doesn't required.